distrib/kotlin_kernel/constants.py

```diff
@@ -0,0 +1,17 @@
+# standard JVM options environment variable
```

Comment:
`constants` is too generic name. Maybe `env_names` or `envs`, what do you think?

Reply:
Build works fine for me now, and I like `env_names`.